website/pages/blog/aws-improvements-Q1-2022.md

```diff
@@ -55,7 +55,7 @@ CloudQuery key use-cases and features:
   }
   ```
 
-  for more information feel free to check out the documentation [here](https://hub.cloudquery.io/providers/cloudquery/aws/latest#multi-account-configuration-aws-organizations)
```

Comment:
Not required due to https://github.com/cloudquery/cloudquery/blob/09ef545a0c9530128bd6960cb54a8dd35e8311da/website/vercel.json#L25 but saves the redirects (to Hub and from Hub back to site)